ssvsigner/cmd/ssv-signer/ssv-signer.go

```diff
@@ -19,10 +17,18 @@ import (
 
 type CLI struct {
 	ListenAddr         string `env:"LISTEN_ADDR" default:":8080" required:""` // TODO: finalize port
-	Web3SignerEndpoint string `env:"WEB3SIGNER_ENDPOINT" required:""`
+	Web3SignerEndpoint string `env:"WEB3SIGNER_ENDPOINT" required:"" name:"web3signer-endpoint"`
```

Comment:
Can you please explain why we need to set a name? IIRC it worked without it, but maybe I'm missing something

Reply:
its based on this `test`: https://github.com/ssvlabs/ssv/blob/0abdb12953a0670a33e089aff1f848d9e747137a/ssvsigner/cmd/ssv-signer/ssv-signer_test.go#L97

basically, without `name` param, it was like: `web-3-singer-endpoint` cli flag, and it looks very unnatural imo